hawkular-alerts-engine/src/test/java/org/hawkular/alerts/engine/CassPersistenceTest.java

```diff
@@ -61,14 +61,7 @@ public static void initSessionAndResetTestSchema() throws Exception {
 
         keyspace = AlertProperties.getProperty("hawkular-alerts.cassandra-keyspace", "hawkular_alerts_test");
 
-        Session session = CassCluster.getSession();
-
-        // try and clean up if this was somehow left around by a prior run
-        try {
-            session.execute("DROP KEYSPACE " + keyspace);
-        } catch (Throwable t) {
-            // never mind, it may not be there
-        }
+        Session session = CassCluster.getSession(true);
```

Comment:
Not, only it is dropped on initialization.
It means that once the keyspace is initialized and cached is not replaced.
Perhaps I can re-arrange this to be more clear, I admit it that only drop on initialization might be not clear enough.